junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTestExtension.java

```diff
@@ -92,11 +94,19 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
 					return createInvocationContext(formatter, methodContext, arguments, invocationCount.intValue());
 				})
 				.onClose(() ->
-						Preconditions.condition(invocationCount.get() > 0,
+						Preconditions.condition(invocationCount.get() > 0 || ignoreZeroInvocations,
 								"Configuration error: You must configure at least one set of arguments for this @ParameterizedTest"));
 		// @formatter:on
 	}
 
+	@Override
+	public boolean mayReturnEmptyInvocationContext(ExtensionContext extensionContext) {
+		Method templateMethod = extensionContext.getRequiredTestMethod();
+		return findAnnotation(templateMethod, ParameterizedTest.class).map(
+			parameterizedTest -> !parameterizedTest.requireArguments()).orElse(
+				TestTemplateInvocationContextProvider.super.mayReturnEmptyInvocationContext(extensionContext));
```

Comment:
```suggestion
		return findAnnotation(templateMethod, ParameterizedTest.class) //
				.map(parameterizedTest -> !parameterizedTest.requireArguments()) //
				.orElse(false);
```

Reply:
updated

Reply:
> Out of curiosity: where does this format with trailing empty comment come from? (I see it is required by Spotless)

I can speak on this as a former Spotless contributor!

It simply splits long lines into multiple shorter lines. It's a common trick with a few formatters like Eclipse and google-java-format. It works because these formatters leave the empty comments alone.